packages/server/postgres/queries/src/getUserIdsToPauseQuery.sql

```diff
@@ -0,0 +1,5 @@
+/*
+  @name getUserIdsToPauseQuery
+*/
+SELECT id FROM "User"
+WHERE inactive = false AND ("lastSeenAt" IS NULL OR "lastSeenAt" <= :activeThreshold);
```

Comment:
ah! this reminds me, we should probably make `lastSeenAt` non-null.
Out of scope for this PR, but will backlog it.